cmd/evm/internal/t8ntool/transaction.go

```diff
@@ -45,10 +46,10 @@ type result struct {
 // MarshalJSON marshals as JSON with a hash.
 func (r *result) MarshalJSON() ([]byte, error) {
 	type xx struct {
-		Error        string          `json:"error,omitempty"`
-		Address      *common.Address `json:"address,omitempty"`
-		Hash         *common.Hash    `json:"hash,omitempty"`
-		IntrinsicGas uint64          `json:"intrinsicGas,omitempty"`
+		Error        string              `json:"error,omitempty"`
+		Address      *common.Address     `json:"address,omitempty"`
+		Hash         *common.Hash        `json:"hash,omitempty"`
+		IntrinsicGas math.HexOrDecimal64 `json:"intrinsicGas,omitempty"`
```

Comment:
I think `hexutil.Uint64` would be better, since that's the package we want to converge onto using.

Reply:
Thanks, fixed